### ISSUE TITLE
Missions inside Workstations

### ISSUE THREAD

**Issue by WerHai** [War3zuk/War3zuk-Alpha-18-Stable-ModLets#9]
After you build a for example HD Auger inside the HD workbench it doesnt count as mission finished

**Comment by War3zuk** [comment]
Yes it Does, You need to be in the Bench when it completes as youve had to do since Since Alpha 1.1
